draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1092,7 +1096,7 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
             depends on the sender policy for retransmission.
             On entry to the Safe Retreat Phase, the CWND can be
             significantly reduced, when there was multiple loss,
-            a sender recovering all lost data could take multiple RTTs to complete.</t>
+            a sender recovering all lost data could then take multiple RTTs to complete.</t>
```

Comment:
Break into 2 sentences: 

On entry to the Safe Retreat Phase, the CWND can be significantly reduced. In cases with multiple loss, a sender recovering all lost data could then take multiple RTTs to complete.